[2025-04-17T07:30:02.001+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-17T07:30:02.024+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.extract_raw_data scheduled__2025-04-17T07:29:00+00:00 [queued]>
[2025-04-17T07:30:02.036+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.extract_raw_data scheduled__2025-04-17T07:29:00+00:00 [queued]>
[2025-04-17T07:30:02.037+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-04-17T07:30:02.056+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): extract_raw_data> on 2025-04-17 07:29:00+00:00
[2025-04-17T07:30:02.061+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=12505) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-17T07:30:02.063+0000] {standard_task_runner.py:72} INFO - Started process 12510 to run task
[2025-04-17T07:30:02.063+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extract_raw_data_pipeline', 'extract_raw_data', 'scheduled__2025-04-17T07:29:00+00:00', '--job-id', '175', '--raw', '--subdir', 'DAGS_FOLDER/lead_data_pipeline.py', '--cfg-path', '/tmp/tmp6wmvu7l4']
[2025-04-17T07:30:02.064+0000] {standard_task_runner.py:105} INFO - Job 175: Subtask extract_raw_data
[2025-04-17T07:30:02.112+0000] {task_command.py:467} INFO - Running <TaskInstance: extract_raw_data_pipeline.extract_raw_data scheduled__2025-04-17T07:29:00+00:00 [running]> on host c0ac39583bd8
[2025-04-17T07:30:02.204+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_raw_data_pipeline' AIRFLOW_CTX_TASK_ID='extract_raw_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-17T07:29:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-17T07:29:00+00:00'
[2025-04-17T07:30:02.205+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-17T07:30:02.222+0000] {lead_data_pipeline.py:146} INFO - Started batch processing for 2025-04-17.
[2025-04-17T07:30:02.223+0000] {lead_data_pipeline.py:35} INFO - Created synthetic data for RO country code.
[2025-04-17T07:30:02.453+0000] {lead_data_pipeline.py:103} INFO - Written 561 records to the CSV file.
[2025-04-17T07:30:03.536+0000] {lead_data_pipeline.py:119} INFO - Added UUID to the dataset.
[2025-04-17T07:30:03.807+0000] {lead_data_pipeline.py:138} INFO - Updated accessed timestamp.
[2025-04-17T07:30:03.808+0000] {lead_data_pipeline.py:154} INFO - Finished batch processing 2025-04-17.
[2025-04-17T07:30:03.809+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-17T07:30:03.824+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-17T07:30:03.825+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=extract_raw_data_pipeline, task_id=extract_raw_data, run_id=scheduled__2025-04-17T07:29:00+00:00, execution_date=20250417T072900, start_date=20250417T073002, end_date=20250417T073003
[2025-04-17T07:30:03.893+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-17T07:30:03.921+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-17T07:30:03.927+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
